pkg/config/nodetreemodel/read_config_file.go

```diff
@@ -76,10 +83,14 @@ func (c *ntmConfig) readInConfig(filePath string) error {
 
 func (c *ntmConfig) readConfigurationContent(target InnerNode, content []byte) error {
 	var obj map[string]interface{}
-	if err := yaml.Unmarshal(content, &obj); err != nil {
-		return err
+
+	if strictErr := yaml.UnmarshalStrict(content, &obj); strictErr != nil {
+		log.Errorf("warning reading config file: %v\n", strictErr)
+		if err := yaml.UnmarshalStrict(content, &obj); err != nil {
```

Comment:
Why call this a second time? Is it supposed to be calling `yaml.Unmarshal` instead?